package org.hemit.domain.model.exceptions

class MaximumNumberOfParticipantsReachedException : Exception()
